src/api/spec/models/kiwi/image_spec.rb

```diff
@@ -0,0 +1,76 @@
+require 'rails_helper'
+
+RSpec.describe Kiwi::Image, type: :model do
+  include_context 'a kiwi image xml'
+  include_context 'an invalid kiwi image xml'
+
+  describe 'validations' do
```

Comment:
we could have also validated the [relations[(http://www.rubydoc.info/github/thoughtbot/shoulda-matchers/Shoulda%2FMatchers%2FActiveRecord%3Ahave_many) and `accept_nested_attributes_for`.

We don't need to fix this in this PR, but just to have it in mind in the future. :wink: